go/vt/mysqlctl/backupstorage/interface.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package backupstorage contains the interface and file system implementation
+// of the backup system.
+package backupstorage
+
+import (
+	"flag"
+	"io"
+
+	log "github.com/golang/glog"
+)
+
+var (
+	// BackupStorageImplementation is the implementation to use
+	// for BackupStorage. Exported for test purposes.
+	BackupStorageImplementation = flag.String("backup_storage_implementation", "", "which implementation to use for the backup storage feature")
+)
+
+// BackupHandle describes an individual backup.
+type BackupHandle interface {
+	// Bucket is the location of the backup. Will contain keyspace/shard.
+	Bucket() string
+
+	// Name is the individual name of the backup. Will contain
+	// tabletAlias-timestamp.
```

Comment:
Good catch. The current stuff doesn't work, I wanted to use an explicit string as you pointed out, but started out with just time in seconds, and left it there (meaning with timestamps of different lengths, it doesn't work at all right now). Will fix, along with the doc.